### DISCUSSION TITLE
can't find my octernship

### DISCUSSION THREAD

**Discussion by bethel-m** [community/community#53002] [GitHub Education]
I previously applied for appwrite octernship, but now when i open my github student page i cannot find the octernships, but instead the message "Oh oh, looks like you need to contribute a bit more to the GitHub community before you’re eligible for an Octernship. Check out our guide on [how to contribute to open-source projects](https://opensource.guide/how-to-contribute/) to learn how you can make a meaningful impact! Keep up the great work and you’ll be a perfect fit in no time!",
please what do i do, I have been working on my assignment to submit it

**Comment by payalmohnani** [comment]
I had a similar issue where one day I was allowed to apply, while a few days later the same message displayed on the screen, as you mentioned. I just made a few commits in my repository, which I had to do for my other work, and then I was allowed to apply again.

**Reply by bethel-m** [reply]
ok, that is helpful

**Comment by juanpflores** [comment]
Thank you for bringing this to our attention. We've passed this along to our internal team to investigate and when we have any updates we'll update this thread. 👍

**Reply by bethel-m** [reply]
alright thank you
